Tasks/Common/VstsAzureRestHelpers_/VstsAzureRestHelpers_.psm1

```
@@ -168,6 +168,8 @@ function Get-SpnAccessToken {
     }
     catch
     {
+        $exceptionMessage = $_.Exception.Message.ToString()
+        Write-Error "ExceptionMessage: $exceptionMessage (in function: Get-SpnAccessToken)"
```

Should this be Write-Verbose since we seem to hiding the failure message.